packages/app-headless-cms/src/admin/plugins/fieldRenderers/object/multipleObjectsAccordion.tsx

```diff
@@ -83,10 +84,11 @@ const ObjectsRenderer = (props: CmsModelFieldRendererProps) => {
     }
 
     const settings = fieldSettings.getSettings();
+    const open = getAccordionRenderSettings(field).open;
```

Comment:
This is a stylistic thing, but let's use the syntax that we usually use everywhere in our code base:

```ts
const { open } = getAccordionRenderSettings(field);
```